src/common/src/DescriptionList/DescriptionList.test.tsx

```diff
@@ -41,4 +41,9 @@ describe('<DescriptionList />', () => {
 		const { getByText } = render(<DescriptionList list={[...list, { label: '', value: <DummyReactComponent /> }]} />);
 		expect(getByText('__REACT_COMPONENT__')).toBeInTheDocument();
 	});
+
+	it('optionally renders a footer with a React node', () => {
+		render(<DescriptionList list={[...list, { label: '', value: <DummyReactComponent /> }]} footer={<>[FOOTER]</>} />);
+		expect(screen.getByTestId('description-footer').textContent).toEqual('[FOOTER]')
```

Comment:
'screen' was new for me, but looked it up, nice short solution!

Reply:
Yeah, makes it a little easier; no more preparing/keeping a reference in scope :+1: